drivers/soundwire/cadence_master.c

```diff
@@ -1262,15 +1262,7 @@ static const struct sdw_master_port_ops cdns_port_ops = {
  */
 bool sdw_cdns_is_clock_stop(struct sdw_cdns *cdns)
 {
-	u32 status;
-
-	status = cdns_readl(cdns, CDNS_MCP_STAT) & CDNS_MCP_STAT_CLK_STOP;
-	if (status) {
-		dev_dbg(cdns->dev, "Clock is stopped\n");
-		return true;
-	}
-
-	return false;
+	return !!(cdns_readl(cdns, CDNS_MCP_STAT) & CDNS_MCP_STAT_CLK_STOP);
```

Comment:
@plbossart Change is ok, but I'd maybe leave the Titanic reference out from the permanent git log history (maybe leave it as a commit note). But yeah, up to you...

Reply:
This will be squashed so no issues with rants.